app/models/candidate.rb

```diff
@@ -8,6 +8,9 @@ class Candidate < ApplicationRecord
   devise :timeoutable
   audited last_signed_in_at: true
 
+  scope :transaction_emails_filter, -> { where({ submission_blocked: false, account_locked: false }) }
```

Comment:
Not precious about this suggestion
```suggestion
  scope :for_transaction_emails, -> { where({ submission_blocked: false, account_locked: false }) }
```
Would read as `Candidate.for_transaction_emails`